### DISCUSSION TITLE
AI based doumentation search on Servicestack docs

### DISCUSSION THREAD

**Discussion by MBrekhof** [ServiceStack/Discuss#82] [Ideas]
Basically something like demo-ed at https://www.youtube.com/watch?v=Yhtjd7yGGGA however on the servicestack documentation would be nice as an example and helpful at the same time.

**Comment by EvoTechMike** [comment]
Maybe there is an easy way to ingest data into docsgpt? https://github.com/arc53/DocsGPT

That way people can run locally and use their own API keys

**Comment by mythz** [comment]
For anyone exploring AI tools, [llms.txt and llms-full.txt support is now available](https://servicestack.net/posts/llms-txt) for a quick extract of docs into AI tools which support it:

 - Summary: https://docs.servicestack.net/llms.txt
 - Detailed: https://docs.servicestack.net/llms-full.txt

**Comment by EvoTechMike** [comment]
This is perfect!
